.github/workflows/codeql-analysis.yml

```diff
@@ -4,10 +4,11 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
   pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [ master ]
+    branches:
+      - master
```

Comment:
Good point, but because it was set that way for a long time, and didn't make any problems, I'm removing those branch related triggers, and only leaving scheduled check